packages/manager/apps/container/src/cookie-policy/CookiePolicy.spec.tsx

```diff
@@ -0,0 +1,104 @@
+import { render, waitFor } from '@testing-library/react';
+import { it, vi, describe, expect } from 'vitest';
+import { initShell } from '@ovh-ux/shell';
+import CookiePolicy from './CookiePolicy';
+import { Environment, User } from '@ovh-ux/manager-config';
+import ApplicationContext from '@/context/application.context';
+
+const onValidate = vi.fn();
+const trackingInit = vi.fn().mockResolvedValue(undefined);
+const trackingSetEnabled = vi.fn().mockResolvedValue(undefined);
+const trackingOnConsentModalDisplay = vi.fn().mockResolvedValue(undefined);
+
+const renderCookiePolicy = async () => {
+  const shell = await initShell();
+  const environment = shell.getPlugin('environment').getEnvironment();
+  render(
+    <ApplicationContext.Provider value={{ shell, environment }}>
+      <CookiePolicy onValidate={onValidate} shell={shell} />
+    </ApplicationContext.Provider>
+  );
+};
+
+const mockedShell = (region: string) => ({
+  getPlugin: (plugin: string) => ({
+    environment: {
+      getEnvironment: () => ({
+        user: {
+          ovhSubsidiary: region,
+        } as User,
+        getRegion: () => region,
+      } as Environment),
+    },
+    tracking: {
+      setRegion: vi.fn(),
+      init: trackingInit,
+      setEnabled: trackingSetEnabled,
+      onConsentModalDisplay: trackingOnConsentModalDisplay,
+    },
+  }[plugin]),
+});
+
+vi.mock('@ovh-ux/shell', () => ({
+  initShell: vi.fn()
+}));
+
+vi.mock('react-cookie', () => ({
+  useCookies: vi.fn(),
+}));
```

Comment:
```suggestion
vi.mock('@ovh-ux/shell');

vi.mock('react-cookie');
```
Returning specific function isn't necessary